articles/cognitive-services/Labs/Answer-Search/c-sharp-quickstart.md

```diff
@@ -22,7 +22,7 @@ The following C# example creates and sends a query for information about the thi
 
 You will need [Visual Studio 2017 or later](https://www.visualstudio.com/downloads/) to run this code on Windows. (The free Community Edition will work.)
 
-Get an access key for the free trial [Cognitive Services Labs](https://labs.cognitive.microsoft.com/en-us/project-answer-search)
+Get an access key for the free trial [Cognitive Research Technologies](https://labs.cognitive.microsoft.com/en-us/project-answer-search)
```

Comment:
You need to test these links and see where they redirect too.. for example the one above goes to https://www.microsoft.com/en-us/research/project/answer-search/